<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>479</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>742</width>
    <height>479</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>742</width>
    <height>479</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Item selection</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhDigitsOnly</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>301</width>
      <height>101</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>301</width>
      <height>101</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap>title.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>291</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>291</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Payment details&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>291</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>291</width>
      <height>19</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>291</width>
      <height>19</height>
     </size>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="colour_text">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>19</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>151</width>
      <height>19</height>
     </size>
    </property>
    <property name="text">
     <string>Colour:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colour">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>131</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>19</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>151</width>
      <height>19</height>
     </size>
    </property>
    <property name="text">
     <string>Category:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="category">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>291</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>291</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>19</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>151</width>
      <height>19</height>
     </size>
    </property>
    <property name="text">
     <string>Keywords:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="keywords">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>291</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>291</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="add_item">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>112</width>
      <height>34</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>112</width>
      <height>34</height>
     </size>
    </property>
    <property name="text">
     <string>Add item</string>
    </property>
   </widget>
   <widget class="QListWidget" name="item_list">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>291</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>291</width>
      <height>351</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>291</width>
      <height>351</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_item">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>112</width>
      <height>34</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>112</width>
      <height>34</height>
     </size>
    </property>
    <property name="text">
     <string>Delete item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>291</width>
      <height>34</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>291</width>
      <height>34</height>
     </size>
    </property>
    <property name="text">
     <string>GO!</string>
    </property>
   </widget>
   <widget class="QComboBox" name="time">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>291</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>291</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>19</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>151</width>
      <height>19</height>
     </size>
    </property>
    <property name="text">
     <string>Time of drop:</string>
    </property>
   </widget>
   <widget class="QLabel" name="size_text">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>131</width>
      <height>19</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>19</height>
     </size>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="size">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>131</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="size_drop">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>131</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <action name="actionStrict_item_selection">
   <property name="text">
    <string>Strict item selection</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>category</tabstop>
  <tabstop>keywords</tabstop>
  <tabstop>colour</tabstop>
  <tabstop>size_drop</tabstop>
  <tabstop>size</tabstop>
  <tabstop>add_item</tabstop>
  <tabstop>delete_item</tabstop>
  <tabstop>item_list</tabstop>
  <tabstop>start</tabstop>
  <tabstop>time</tabstop>
 </tabstops>
 <resources>
  <include location="../window/Pictures/logo.qrc"/>
 </resources>
 <connections/>
</ui>
